src/WindowManager.vala

```diff
@@ -155,7 +155,6 @@ namespace Gala {
             unowned Meta.Display display = get_display ();
             display.gl_video_memory_purged.connect (() => {
                 Meta.Background.refresh_all ();
-                SystemBackground.refresh ();
```

Comment:
Do we still need this to apply the background color again?

Reply:
`Meta.Background.refresh_all ()` should do it. We had a separate `refresh ()` function because previously there was a noisy texture used in the system background.